docs/design/ovs-pipeline.md

```diff
@@ -620,48 +636,54 @@ non-ARP and non-IP (assuming any can be received by the switch) is actually
+K8s NetworkPolicy ->  IngressRuleTable(90)
+All other Tiers   ->  AntreaPolicyIngressRuleTable(85)
 ```
 
-Again for these two tables, you will need to keep in mind the ACNP
+Again for this table, you will need to keep in mind the ACNP
 [specification](#antrea-networkpolicy-crd-implementation) that we are using.
 Since the example ACNP resides in the Application tier, if you dump the flows
-for table 89, you should see something like this:
+for table 85, you should see something like this:
 
 ```text
 1. table=89, priority=64990,ct_state=-new+est,ip actions=resubmit(,105)
-2. table=89, priority=58600,conj_id=1,ip actions=drop
-3. table=89, priority=58600,ip,nw_src=10.10.1.7 actions=conjunction(1,1/3)
-4. table=89, priority=58600,ip,reg1=0x19c actions=conjunction(1,2/3)
-5. table=89, priority=58600,tcp,tp_dst=80 actions=conjunction(1,3/3)
+2. table=89, priority=14000,conj_id=4,ip actions=load:0x4->NXM_NX_REG3[],load:0x1->NXM_NX_REG0[20],resubmit(,101)
+3. table=89, priority=14000,ip,nw_src=10.10.1.7 actions=conjunction(4,1/3)
+4. table=89, priority=14000,ip,reg1=0x19c actions=conjunction(4,2/3)
+5. table=89, priority=14000,tcp,tp_dst=80 actions=conjunction(4,3/3)
 6. table=89, priority=0 actions=resubmit(,90)
 ```
 
-As for [AntreaPolicyEgressRuleTables], flow 1 (highest priority) ensures that for
-established connections packets go straight to [L2ForwardingOutTable],
-with no other match required.
+As for [AntreaPolicyEgressRuleTable], flow 1 (highest priority) ensures that for
+established connections packets go straight to IngressMetricsTable,
+then [L2ForwardingOutTable], with no other match required.
 
 The rest of the flows read as follows: if the source IP address is in set
 {10.10.1.7}, and the destination OF port is in the set {412} (which
 correspond to IP addresses {10.10.1.6}), and the destination TCP port
-is in the set {80}, then use `conjunction` action with id 1, which drops the
-packet. Otherwise, go to the next AntreaPolicyRuleTable belonging to the next
-tier in precedence, or in the case of the Application tier (lowest precedence),
-go to the [IngressRuleTable]. One notable difference is how we use OF ports to
-identify the destination of the traffic, while we use IP addresses in
-[AntreaPolicyEgressRuleTables] to identify the source of the traffic. More details
-regarding this can be found in the following [IngressRuleTable] section.
-
-As seen in [AntreaPolicyEgressRuleTables], the default action is to evaluate K8s
+is in the set {80}, then use `conjunction` action with id 4, which loads
+the `conj_id` 4 into NXM_NX_REG3, a register used by Antrea internally to
+indicate the disposition of the packet is Drop, and forward the packet to
+IngressMetricsTable for it to be dropped.
+
+Otherwise, go to the next AntreaPolicyRuleTable belonging to the next
+tier in precedence, or in the case of the Application tier (lowest precedence
+before K8s NetworkPolicies), go to the [IngressRuleTable].
```

Comment:
is this information up-to-date? we have a single table for all tiers now

Reply:
Thanks for the catch... updated